extensions/wikia/PageShare/scripts/PageShare.js

```diff
@@ -72,8 +62,40 @@ require(['wikia.window', 'wikia.tracker', 'jquery'], function(win, tracker, $) {
 		});
 	}
 
-	// bind events to links
-	$(function() {
-		loadShareIcons();
-	});
+	/**
+	 * @desc Returns a language code.
+	 * If user is logged in, language code is read from user's preferences.
+	 * For anonymous users first two letters of the browser/system regional preferences are used.
+	 * Both values are ignored and overwritten if function is provided with a non-false parameter.
+	 *
+	 * @param {*} useLangQueryStringParam
+	 * @returns {String|null}
+	 */
+	function getShareLang(useLangQueryStringParam) {
+		// forced by param
+		if (useLangQueryStringParam) {
+			return useLangQueryStringParam;
+		// logged in user
+		} else if (win.wgUserName) {
+			return win.wgUserLanguage;
+		// anonymous user who uses
+		// Chrome or Firefox
+		} else if (win.navigator.languages) {
+			return win.navigator.languages[0].substr(0, 2);
+		// Safari
+		} else if (win.navigator.language) {
+			return win.navigator.language.substr(0, 2);
+		// Internet Exploder
+		} else if (win.navigator.userLanguage) {
+			return win.navigator.userLanguage.substr(0, 2);
+		// something exotic
+		} else {
+			return null;
```

Comment:
we can be extra fancy here and use wikia.log to log language.
Example usage:
```javascript
wikia.log([message], 'debug', page_share)
```
First argument is an array of messages which are concatenated with ","
Second argument is log level (debug in this case is enough)
Third argument is log group.

This way you can debug which language is chosen by appending to URL:
?log_level=9&log_group=pageShare